third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -371,8 +373,13 @@ impl<'a> TransferFunctions for LiveVarAnalysis<'a> {
             Branch(id, _, _, src) => {
                 state.insert_or_update(*src, self.livevar_info(id, offset), self.track_all_usages);
             },
-            Prop(id, _, exp) => {
-                for (idx, _) in exp.used_temporaries(self.func_target.global_env()) {
+            Prop(id, _, exp) if self.track_all_usages => {
+                for idx in exp.used_temporaries() {
+                    state.livevars.insert(idx, self.livevar_info(id, offset));
```

Comment:
I think this should be `state.insert_or_update` so that we join rather than replace (which is what `insert` would do)?

Reply:
Done